mcp-server/assets/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```

Comment:
Let's remove this?

Reply:
if we keep, can we make it relevant to this particular app?  looks like its a basic messaging template